lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -64,10 +68,20 @@ class CourseEnrollmentSerializer(serializers.ModelSerializer):
     Serializes CourseEnrollment models
     """
     course = CourseField()
+    certificate = serializers.SerializerMethodField('get_certificate')
+
+    def get_certificate(self, model):
+        certificate_info = certificate_status_for_student(model.user, model.course_id)
+        if certificate_info['status'] == CertificateStatuses.downloadable:
+            return {
+                "url": certificate_info['download_url'],
+            }
+        else:
+            return {}
```

Comment:
In the future, we may include other certificate fields, even if the user hasn't yet earned their certificate.  So for forward compatibility, I chose to return an empty dict for now.